lib/tasks/boostrap.rake

```diff
@@ -10,7 +10,7 @@ namespace :db do
     end
   end
 
-  desc 'Create dummy users'
+  desc 'Create dummy user'
```

Comment:
I actually liked this title

Reply:
I was getting a bunch of failed tests and wasn't sure how to fix it, but changing it back made it work.

Reply:
If it fails next times, let me know and we will fix together ;)